apps/pollbook/frontend/src/app_poll_worker_screen.test.tsx

```diff
@@ -281,7 +259,9 @@ describe('PollWorkerScreen', () => {
     act(() => {
       vi.advanceTimersByTime(DEFAULT_QUERY_REFETCH_INTERVAL);
     });
-    const firstRow = await screen.findByTestId('voter-row#123');
+    const firstRow = await vi.waitFor(() =>
+      screen.getByTestId('voter-row#123')
+    );
```

Comment:
Saw this test flake on my branch. IIRC in the past `vi.waitFor` has been more reliable so trying that out here.